king_phisher/client/widget/managers.py

```diff
@@ -373,3 +374,61 @@ def signal_activate_popup_menu_copy(self, menuitem, column_ids):
 
 	def signal_activate_popup_menu_delete(self, menuitem):
 		self._call_cb_delete()
+
+class TimeSelector(gui_utilities.GladeGObject):
+	dependencies = gui_utilities.GladeDependencies(
+		children=(
+			'spinbutton_hour',
+			'spinbutton_minute'
+		),
+		top_level=(
+			'ClockHourAdjustment',
+			'ClockMinuteAdjustment'
+		)
+	)
+	top_gobject = 'popover'
+
+class TimeSelectorButtonManager(object):
+	def __init__(self, button, application, value=None):
+		"""
```

Comment:
I would add a description here of what this class is used for. The parameter descriptions are good.